nuget/lib/dependabot/nuget/nuget_config_credential_helpers.rb

```diff
@@ -0,0 +1,70 @@
+# typed: true
+# frozen_string_literal: true
+
+module Dependabot
+  module Nuget
+    module NuGetConfigCredentialHelpers
+      def self.user_nuget_config_path
+        home_directory = Dir.home
+        File.join(home_directory, ".nuget", "NuGet", "NuGet.Config")
+      end
+
+      def self.temporary_nuget_config_path
+        user_nuget_config_path + "_ORIGINAL"
```

Comment:
should this be a "random" name or prefix in case a previous run fail and didn't rename the file back?

Reply:
It shouldn't matter.  When actually running, we'll only use this container for a single job.  I considered patching this file as a part of the job starting, but then it would have been impossible to test, so I instead use the wrapper below.